README.md

```diff
@@ -39,6 +39,27 @@ Can be sourced from jcenter like so:
       <classifier>sources|tests|javadoc|all</classifier> (Optional)
     </dependency>
 	
+**Note**: You need to have bintray maven repository configured to be able to pull bitbucket-rest dependency.
```

Comment:
Lets remove this line as that's already made clear further up above.

Reply:
Agree, removed already